CHANGELOG.md

```diff
@@ -2516,4 +2537,4 @@ Beginning with this release, the navigation SDK and Core Navigation are written
 
 ## v0.0.2 (January 19, 2017)
 
-Initial public release
\ No newline at end of file
+Initial public release
```

Comment:
What's this change?

Reply:
Xcode added a new line at the end of this file.